specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,43 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/appTemplates": {
+      "get": {
+        "tags": [
+          "Apps"
+        ],
+        "description": "Get all available application templates.",
+        "operationId": "Apps_Templates",
```

Comment:
nit: "App_Templates"

Reply:
Same for file naming

Reply:
yes, but I'm trying to follow other naming convention, Apps_xxx